packages/select/src/containers/SelectContainer.js

```diff
@@ -15,7 +15,7 @@ export default class SelectContainer extends ControlledComponent {
     /**
      * Appends the menu to the provided element
      */
-    appendToNode: PropTypes.instanceOf(Element),
+    appendToNode: PropTypes.any,
```

Comment:
Nice, this check never actually worked most of the time. So good to just be super flexible in what it accepts.